Testing/VelaptorTests/OpenGL/GLWindowTests.cs

```diff
@@ -786,10 +792,11 @@ public void Initialized_WhenWindowIsInitialized_ReturnsTrue()
     public void TypeOfBorder_WhenGettingInvalidValue_ThrowsException()
     {
         // Arrange
-        var expected = $"The value of the enum '{nameof(Silk.NET.Windowing.WindowBorder)}' used in the class '{nameof(GLWindow)}' and";
-        expected += " method 'SetupOtherPropCaches' is invalid and out of range.";
+        var invalidValue = 1234;
```

Comment:
Please change to a constant

```suggestion
        const invalidValue = 1234;
```